pom.xml

```diff
@@ -1,27 +1,25 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
-
     <parent>
-        <groupId>org.jenkins-ci.plugins</groupId>
-        <artifactId>plugin</artifactId>
-        <version>2.11</version>
-        <relativePath />
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>plugin</artifactId>
+      <version>3.43</version> <!-- or whatever the newest version available is -->
+      <relativePath />
     </parent>
 
     <artifactId>argus-notifier</artifactId>
-    <version>1.0.6-SNAPSHOT</version>
+    <version>1.0.7-SNAPSHOT</version>
     <packaging>hpi</packaging>
 
     <properties>
         <!-- Baseline Jenkins version you use to build the plugin. Users must have this version or newer to run. -->
-        <jenkins.version>2.60</jenkins.version>
+        <jenkins.version>2.164.3</jenkins.version>
```

Comment:
Please change this to out current supported version `2.164.2.1`

Reply:
done

Reply:
Just an FYI, you want to be careful about how far you bump these versions in case folks in the open-source community haven't updated. In this case, this should be fine since this version has been around since May.